public/index.html

```diff
@@ -30,6 +30,7 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="modal"></div>
```

Comment:
Shouldn't we ideally not alter the html file? it should only have root div within body

Reply:
Actually I am using portal for the modal and that's why I am doing like this